bugs_to_be_fixed.txt

```diff
@@ -5,12 +5,7 @@ need to have it execute either on same turn or before next turn
 **fixed, needs code clean up with potential redundencies
 
 2. When placing ships we need to have a verification for computer that it placed 2 ships
-(in case it cancels due to overlap)
+(in case it cancels due to overlap)***Stephen - soft fix
```

Comment:
I still don't understand how the computer is getting around all of our parameters! Definitely going to ask Abdul for advice on that one